build/akri-containers.mk

```diff
@@ -51,6 +51,10 @@ $(eval $(call add_rust_targets,controller,controller))
 $(eval $(call add_rust_targets,agent,agent))
 $(eval $(call add_rust_targets,udev,udev-video-broker))
 $(eval $(call add_rust_targets,webhook-configuration,webhook-configuration))
+$(eval $(call add_rust_targets,debug-echo-discovery,debug-echo-discovery))
```

Comment:
Probably add new things to `akri-docker-all`

Reply:
Added in 34e97dcf0